### ISSUE TITLE
Why it said that it's "forces replacement" while i change the "internet_max_bandwidth_out".

### ISSUE THREAD

**Issue by hanyouqing** [tencentcloudstack/terraform-provider-tencentcloud#498]
Issues on GitHub are intended to be related to bugs or feature requests with provider codebase,
so we recommend using our other community resources instead of asking here 👍.

---

When I change `internet_max_bandwidth_out` in resource `tencentcloud_instance`, it just said it'll create a new instance which is not expected. 

I just want to change the bandwidth of the instance, and applications are running on this instance, any other options?

```
-/+ resource "tencentcloud_instance" "xxx-xxx-inst" {
        allocate_public_ip                      = true
        availability_zone                       = "na-siliconvalley-2"
      ~ create_time                             = "2020-08-04T06:20:27Z" -> (known after apply)
        disable_monitor_service                 = false
        disable_security_service                = false
      ~ expired_time                            = "2020-09-04T06:20:27Z" -> (known after apply)
        force_delete                            = true
        hostname                                = "xxx"
      ~ id                                      = "ins-xxx" -> (known after apply)
        image_id                                = "img-pi0ii46r"
        instance_charge_type                    = "PREPAID"
        instance_charge_type_prepaid_period     = 1
        instance_charge_type_prepaid_renew_flag = "NOTIFY_AND_AUTO_RENEW"
        instance_name                           = "xxxxxx"
      ~ instance_status                         = "RUNNING" -> (known after apply)
        instance_type                           = "S3.SMALL1"
        internet_charge_type                    = "TRAFFIC_POSTPAID_BY_HOUR"
      ~ internet_max_bandwidth_out              = 20 -> 40 # forces replacement
        key_name                                = "skey-ccycgz09"
      ~ private_ip                              = "" -> (known after apply)
```

**Comment by piggona** [comment]
Hi @hanyouqing 
Feature supported for version 1.41.1 and after:
`adjusting internet_max_bandwidth_out without forceNew when attribute internet_charge_type within TRAFFIC_POSTPAID_BY_HOUR,BANDWIDTH_POSTPAID_BY_HOUR,BANDWIDTH_PACKAGE`

**Comment by hanyouqing** [comment]
@piggona Thanks!
This has issue been fixed on `1.41.2`, can be fixed with version specified.

```
provider "tencentcloud" {
  region = var.xxxx.beijing.region
  version = ">= v1.41.2"
}
```
